### PR TITLE
some cleanup

### PR DESCRIPTION
**Pull request by xu-cheng**

### CODE REVIEW

File: lib/bundle/brew_dumper.rb [hunk 1]
```diff
@@ -69,18 +73,18 @@ def formula_inspector f
...
-        args: args,
-        version: keg["version"],
-        dependencies: f["dependencies"],
-        requirements: f["requirements"],
+        :name => f["name"],
+        :full_name => f["full_name"],
+        :args => args,
+        :version => keg["version"],
+        :dependencies => f["dependencies"],
+        :requirements => f["requirements"],
```
Comment (mikemcquaid) [inline lib/bundle/brew_dumper.rb]: Why, out of interest? I prefer the existing style.

**Comment by mikemcquaid** [comment]
Could we get a Rubocop to handle this automatically in future?

File: lib/bundle/brew_dumper.rb (hunk 1 above)
Comment (xu-cheng) [inline lib/bundle/brew_dumper.rb]: This is changed by Rubocop(Using `brew style --fix <path to homebrew/bundle>`). Maybe we can change the rubocop rule if you want.

Comment (mikemcquaid) [inline lib/bundle/brew_dumper.rb]: Ok, cool. I guess it's better to be consistent for now.

**Pull request merged**